### ISSUE TITLE
(FE) 10-10EZR Bug: Do Not allow users to use hyphens in phone number

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#74329]
- [ ] Currently: users can use hyphens in their phone number but VES/API cannot accept them
- [ ] This came up in EZR bug on Jan 23, 2024 at 6:12:32PM UTC
- [ ] Desired state: Do Not allow users to enter hyphens or other characters besides numbers in phone number fields
- [ ] If not already there: Have helper text that says something to the effect of "please enter numbers only" (work with Alex and UX on this text if needed)

**Comment by mkerns1** [comment]
Path Forward Per Demo discussion:
Format the phone number field to show hyphenated phone number for the user while stripping hyphen out on submission.

**Comment by mkerns1** [comment]
Progress blocked by ticket [#74752](https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/74752)

**Comment by mkerns1** [comment]
### Current Update on Phone Number Input Validation

**Progress Overview:**

I have successfully updated the phone number input fields across the application to limit user input to numeric characters only. This change addresses the issues identified in the EZR bug reported on January 23, 2024.

**Key Achievements:**

- [ ] Numeric Character Restriction: The phone number fields now restrict input to numbers only. This aligns with the requirements of the VES/API, which cannot process non-numeric characters such as hyphens.
- [ ] Front-End Validation Implemented: I've added validation checks to ensure users can only enter numeric characters. This prevents the submission of phone numbers with hyphens or other invalid characters.
- [ ] User-Friendly Helper Text: In collaboration with Alex and the UX team, I've introduced helper text adjacent to the phone number fields. The text reads "please enter numbers only," guiding users on the expected format.

**Next Steps:**

Final Review and QA: I will be conducting a thorough review and quality assurance to ensure the changes meet our acceptance criteria. Collaboration with the UX team will continue to refine the user interface.

**Action Items:**

Team members are encouraged to review the changes and provide feedback.
QA testing schedules will be shared shortly for team coordination.

**Link to PR:**

GitHub PR for reference: [department-of-veterans-affairs/va.gov-team#74329](https://github.com/department-of-veterans-affairs/vets-website/pull/27755)

**Comment by mkerns1** [comment]
Current Update on Phone Number Input Helper Text:

Per Aliyah:

**Comment by mkerns1** [comment]
PR has been merged into the main branch.

Link to PRs [here](https://github.com/department-of-veterans-affairs/vets-website/pull/27755) and [here](https://github.com/department-of-veterans-affairs/vets-website/pull/27769).
